### PR TITLE
zh-CN: TensorFlow Lite guide get_started.md

### PR DESCRIPTION
**Pull request by logiccodes**
Translated `get_started.md`.
Already signed `individual CLA`.

### CODE REVIEW

File: site/zh-cn/lite/guide/get_started.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
+# 开始使用TensorFlow Lite
+
+TensorFlow Lite提供了转换TensorFlow模型，并在移动端(mobile)、嵌入式(embeded)和物联网(IoT)设备上运行TensorFlow模型所需的所有工具。以下指南将逐步介绍开发人员各环节的工作流程，并给出了指向更多说明的链接。
```
Comment (Mr-Linus) [inline site/zh-cn/lite/guide/get_started.md]: 中英文之间请使用空格分隔，以下改不赘述

File: site/zh-cn/lite/guide/get_started.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+## 1. 选择一个模型
+
+<a id="1_choose_a_model"></a>
+
+TensorFlow Lite允许您在多种设备上运行TensorFlow模型。TensorFlow模型是一种数据结构，这种数据结构包含了在解决一个特定问题时，训练得到的机器学习网络的逻辑和知识。
```
Comment (Mr-Linus) [inline site/zh-cn/lite/guide/get_started.md]:
```suggestion
TensorFlow Lite 允许您在多种设备上运行 TensorFlow 模型。TensorFlow 模型是一种数据结构，这种数据结构囊括了经过训练并用于解决特定问题的机器学习网络的逻辑和知识。
```

File: site/zh-cn/lite/guide/get_started.md (hunk 1 above)
Comment (logiccodes) [reply site/zh-cn/lite/guide/get_started.md]: @Mr-Linus 已经按”格式标准、术语比对、审校重点“文档进行了修改，增加了英文字符和中文字符间的空格（标点符号相邻的英文除外），并将英文注释的全部括号换成了中文全角。

Comment (logiccodes) [reply site/zh-cn/lite/guide/get_started.md]: 这段已修改成
TensorFlow Lite 提供了转换 TensorFlow 模型，并在移动端（mobile）、嵌入式（embeded）和物联网（IoT）设备上运行 TensorFlow 模型所需的所有工具。以下指南将逐步介绍开发人员各环节的工作流程，并给出了指向更多说明的链接。

File: site/zh-cn/lite/guide/get_started.md (hunk 2 above)
Comment (logiccodes) [reply site/zh-cn/lite/guide/get_started.md]: 已修改

File: site/zh-cn/lite/guide/get_started.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+<a id="3_use_the_tensorflow_lite_model_for_inference_in_a_mobile_app"></a>
+
+*推理（Inference）* 是通过模型（model）运行数据（data）以获得预测（predictions）的过程。这个过程需要模型（model）、解释器（interpreter）和输入数据（input data）。
+
+### TensorFlow Lite 解释器
+
+[TensorFlow Lite 解释器（interpreter）](inference.md)是一个库（library），它接收一个模型文件（model file），执行模型文件在输入数据（input data）上定义的运算符（operations），并提供对输出（output）的访问。
+
+该解释器（interpreter）适用于多个平台，提供了一个简单的 API，用于从 Java, Swift, Objective-C, C++ 和 Python 运行 TensorFlow Lite 模型。
```
Comment (Mr-Linus) [inline site/zh-cn/lite/guide/get_started.md]:
```suggestion
该解释器（interpreter）适用于多个平台，提供了一个简单的 API，用于从 Java、Swift、Objective-C、C++ 和 Python 运行 TensorFlow Lite 模型。
```

File: site/zh-cn/lite/guide/get_started.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+
+```java
+try (Interpreter interpreter = new Interpreter(tensorflow_lite_model_file)) {
+  interpreter.run(input, output);
+}
+```
+
+### GPU 加速和委托
+
+一些设备为机器学习运算符提供硬件加速（hardware acceleration）。例如，大多数移动电话有 GPUs，GPU 可以比 CPU 执行更快的浮点矩阵运算（floating point matrix operations）。
```
Comment (Mr-Linus) [inline site/zh-cn/lite/guide/get_started.md]:
```suggestion
一些设备为机器学习运算符提供硬件加速（hardware acceleration）。例如，大多数手机有 GPU，这些 GPU 可以比 CPU 执行更快的浮点矩阵运算（floating point matrix operations）。
```

File: site/zh-cn/lite/guide/get_started.md [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+Interpreter.Options options = (new Interpreter.Options()).addDelegate(delegate);
+Interpreter interpreter = new Interpreter(tensorflow_lite_model_file, options);
+try {
+  interpreter.run(input, output);
+}
+```
+
+要添加对新硬件加速器的支持，您可以[定义您自己的委托](../performance/delegates.md#how_to_add_a_delegate)。
+
+### Android and iOS
```
Comment (Mr-Linus) [inline site/zh-cn/lite/guide/get_started.md]:
```suggestion
### Android 和 iOS
```

Comment (logiccodes) [reply site/zh-cn/lite/guide/get_started.md]: @Mr-Linus 谢谢反馈！已将你的修改建议commit到了我自己的分支，但Google Bot反馈CLA有问题，之前我已经签署了CLA，你可以按下面的提示查看一下CLA的签署情况。

**Comment by Mr-Linus** [comment]
Confirmed the CLAs

**Comment by logiccodes** [comment]
I signed it!
Already signed individual CLA.

**Comment by logiccodes** [comment]
@Mr-Linus As the googlebot suggests "maybe you used a different email address in the git commits than was used to sign the CLA"? As the committed history shows it's not a Gmail account. 

Maybe if we use ssh verification to push the code, which is what I have done, googlebot will only check the GitHub account (right now I've already used my GitHub account email address to sign another CLA), but if we use the GitHub default way to commit the email account is also counted.

So would you please try to **add your GitHub account email** to your signed CLA? And If everything is done, I'll add another comment to let the googlebot run again to check CLA. Thanks!

**Comment by Mr-Linus** [comment]
@logiccodes I have already signed the CLAs.
Maybe I forgot to update my new email address. Thank you for your reminder!

**Comment by Mr-Linus** [comment]
Already signed individual CLA.
Confirmed the CLAs

**Comment by Mr-Linus** [comment]
Update CLAs.

**Comment by Mr-Linus** [comment]
Update CLAs.

**Comment by Mr-Linus** [comment]
@logiccodes Need you to comment again. Thx.

**Comment by logiccodes** [comment]
All right, we've already signed CLAs.
Comment test.

**Comment by logiccodes** [comment]
@googlebot rescan

**Pull request merged**